modules/osdk-create-cr.adoc

```diff
@@ -200,6 +200,13 @@ memcached-sample                        5/5     5            5           3m
 ----
 endif::[]
 
+. Delete the `nginx` CR by running the following command:
++
+[source,terminal]
+----
+$ oc delete -f config/samples/demo_v1alpha1_nginx.yaml
```

Comment:
Better to change to bellow

. Delete the CR:
+
[source,terminal,subs="attributes+"]
----
$ oc delete -f config/samples/{group}_v1_{app}.yaml
----

Reply:
Addressed